content/locales/TRANSLATIONS.md

```diff
@@ -0,0 +1,345 @@
+<a href="https://github.com/TryGhost/Ghost"><img src="https://cloud.githubusercontent.com/assets/120485/18661790/cf942eda-7f17-11e6-9eb6-9c65bfc2abd8.png" alt="Ghost" /></a>
```

Comment:
The tutorial will be ported over to themes.ghost.org before merging 👍 

Maybe a note to add is that you have to restart your server if you change any locales in the theme.

Reply:
Completed the help file, adding subsections on reactivating the edited translation file or theme, etc.